src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -112,7 +112,7 @@ class FacetSidebar extends React.Component {
         const selectedValue = this.state[field] ? this.state[field].id : '';
 
         return (
-          <div className="nypl-searchable-field">
+          <div key={`${facet.field}-${facet.value}`} className="nypl-searchable-field">
```

Comment:
This repo is using ES6 and string interpolation is finally added to Javascript in ES6 in the form of
```
`${variableName} and the rest of the string`
```
Don't think the back tick was previously used in JS. It does seem verbose with the surrounding braces but that's because it's not a regular string and it needs to be evaluated inside the curly braces.